ffi/derive/tests/ui.rs

```diff
@@ -3,6 +3,7 @@ use trybuild::TestCases;
 #[test]
 fn ui() {
     let test_cases = TestCases::new();
-    test_cases.pass("tests/ui_pass/*.rs");
+    test_cases.pass("tests/ui_pass/valid.rs");
+    #[cfg(not(feature = "client"))]
```

Comment:
Is it temporary?

Reply:
most likely, at least there should be equivalent for `client` feature

Reply:
Let's add a TODO then